docs/ixctl/Add-a-Member.md

```diff
@@ -1,5 +1,10 @@
 # Add a Member
 
+Prefix lists and AS-SETs are populated from the IRRs. The AS-SET names come from PeeringDB by default. These can be overridden in ixCtl but it is best to update them in PeeringDB. 
+Before adding a member to ixCtl, ensure the member has:
```

Comment:
You need to have a blank line after this line if you want to have what's under it display as a bulleted list. @sunbuch

Reply:
@jason20c I appreciate your feedback on this. I'm not sure on process now. I made the change and committed it. I don't know where it shows up or if you can see the change to confirm it is now correct.

Reply:
@sunbuch thanks for the update, I can see the change in the files (either from clicking the newer commit message, or the Files Changed tab). Marking this as resolved and merging. Thanks.